ChocolArm64/OpCodeTable.cs

```diff
@@ -17,6 +17,9 @@ static OpCodeTable()
             SetA32("<<<<1010xxxxxxxxxxxxxxxxxxxxxxxx", A32InstInterpret.B,      typeof(A32OpCodeBImmAl));
             SetA32("<<<<1011xxxxxxxxxxxxxxxxxxxxxxxx", A32InstInterpret.Bl,     typeof(A32OpCodeBImmAl));
             SetA32("1111101xxxxxxxxxxxxxxxxxxxxxxxxx", A32InstInterpret.Blx,    typeof(A32OpCodeBImmAl));
+            SetA32("<<<<000100101111111111110011xxxx", A32InstInterpret.Blx,    typeof(A32OpCodeBReg));
+            SetA32("<<<<000100101111111111110001xxxx", A32InstInterpret.Bx,     typeof(A32OpCodeBReg));
+            SetA32("<<<<000100101111111111110010xxxx", A32InstInterpret.Bxj,     typeof(A32OpCodeBReg));
```

Comment:
Alignment issue here:
```suggestion
            SetA32("<<<<000100101111111111110010xxxx", A32InstInterpret.Bxj,    typeof(A32OpCodeBReg));
```